editions/tw5.com/tiddlers/widgets/LetListWidget.tid

```diff
@@ -0,0 +1,26 @@
+title: LetListWidget
+created: 20250307212252946
+modified: 20250307212252946
+tags: Widgets
+caption: letlist
+
+! Introduction
+
+<<.from-version "5.3.7">> The <<.wid letlist>> widget allows multiple variables to be set in one operation, each to the list of results from evaluating an attribute. It is similar to the <<.wlink LetWidget>> widget but assigns the complete list of results to a multi-valued variable instead of just the first result.
+
+! Content and Attributes
+
+The content of the <<.wid letlist>> widget is the scope for the value assigned to the variable.
```

Comment:
@Jermolene -- I think the name for `letlist` is very confusing. I did expect, that it works like the list-widget. So I can access the list variables inside the letlist body. So the variables change similar and in conjunction to the currentTiddler variable. So I was missing the filter-parameter, which would define the number of iterations.